arctic/store/_version_store_utils.py

```diff
@@ -28,9 +30,13 @@ def checksum(symbol, doc):
     Checksum the passed in dictionary
     """
     sha = hashlib.sha1()
-    sha.update(symbol)
-    for k in sorted(doc.iterkeys(), reverse=True):
-        sha.update(str(doc[k]))
+    sha.update(symbol.encode('ascii'))
```

Comment:
Did you verify this produces the same SHA for the `Binary` chunk data?

Not sure that the string should be re-endcoded - can't it just be passed as a byte-array?